src/modules/fancyzones/editor/FancyZonesEditor/FancyZonesEditor.csproj

```diff
@@ -1,193 +1,46 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{5CCC8468-DEC8-4D36-99D4-5C891BEBD481}</ProjectGuid>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <OutputType>WinExe</OutputType>
-    <RootNamespace>FancyZonesEditor</RootNamespace>
-    <AssemblyName>FancyZonesEditor</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <WarningLevel>4</WarningLevel>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
+    <UseWPF>true</UseWPF>
+    
+    <PublishReadyToRun>true</PublishReadyToRun>
+    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
+    <TieredCompilation>true</TieredCompilation>
+
+    <ProductName>FancyZonesEditor</ProductName>
+    <Copyright>Copyright (C) 2019 Microsoft Corp.</Copyright>
+    <Company>Microsoft Corp.</Company>
+    <Version>0.14.1.0</Version>

```

Comment:
It would be great if the version can be imported from common/version.h

Reply:
@enricogior could be probably done another way. that reminds me of [this](https://github.com/microsoft/PowerToys/blob/3095ade94c0d8cbf3672fdb6475f54f8f5a7913c/src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs#L26). We could make a separate msbuild task which generates constants like these. I've created [a separate issue for that.](https://github.com/microsoft/PowerToys/issues/800)